README.md

```diff
@@ -32,6 +32,9 @@ eBPF.
 
 ## Building
 
+The functionality of profiling is planed to become part of the [OTel Collector](https://opentelemetry.io/docs/collector/).
```

Comment:
```suggestion
We are working towards integrating the profiling agent into the [OTel Collector](https://opentelemetry.io/docs/collector/) as a receiver,
which will be the supported configuration going forward. In the meantime, we also offer a standalone profiling agent binary,
to aid with development and debugging. The expectation is that this will go away once the integration with the OTel Collector is complete.
```